src/main/java/com/puppycrawl/tools/checkstyle/api/DetailAST.java

```diff
@@ -146,4 +146,6 @@
      * @return {@code true} if this AST has any children.
      */
     boolean hasChildren();
+
 }
+
```

Comment:
I dont think you need these changes with blank lines :)

Reply:
I added a method but later found a better way, must be a residue of that process, removed them now.